com.github.vikdevelop.photopea_app.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+only-arches: x86_64
+skip-arches: aarch64
+command: photopea_app
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-download
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea_app
+    - install -Dm755 -t /app/share/applications com.github.vikdevelop.photopea_app.desktop
```

Comment:
the build log shows:
`WARNING: Binary not found for Exec line in /srv/buildbot/worker/build-x86_64-7/build/builddir/export/share/applications/com.github.vikdevelop.photopea_app.desktop: /var/lib/flatpak/app/com.github.vikdevelop.photopea_app/x86_64/test/active/files/bin/photopea_app`

because .desktop file is wrong. Exec should be `photopea_app`

(while it's upstream, I don't understand why there are two .desktop file there, and that's probably another source of confusion)

Reply:
@hfiguiere should I also change it from "photopea" to "photopea_app" on line 14?

Reply:
the module name is irrelevant here.